mica-rest/src/main/java/org/obiba/mica/person/rest/PersonsResource.java

```diff
@@ -37,24 +38,28 @@ public PersonsResource(Dtos dtos, PersonService personService, EventBus eventBus
 
   @GET
   @Path("/study/{studyId}")
+  @RequiresPermissions({ "/draft/individual-study:VIEW", "/draft/harmonization-study:VIEW", "/draft/network:VIEW" })
   public Set<PersonDto> getStudyMemberships(@PathParam("studyId") String studyId) {
     return personService.getStudyMemberships(studyId).stream().map(member -> dtos.asDto(member, true)).collect(Collectors.toSet());
   }
 
   @GET
   @Path("/network/{networkId}")
+  @RequiresPermissions({ "/draft/individual-study:VIEW", "/draft/harmonization-study:VIEW", "/draft/network:VIEW" })
```

Comment:
You need to be more specific, with the networkId, because mica allows to have specific permissions on the draft documents.